core/watcher/eventsource/namespace.go

```diff
@@ -61,7 +61,9 @@ func (w *NamespaceWatcher) loop() error {
 	}
 	subscription, err := w.watchableDB.Subscribe(changestream.Namespace(w.namespace, w.changeMask))
 	if err != nil {
-		return errors.Annotatef(err, "subscribing to namespace %q", w.namespace)
+		// TODO(wallyworld) - remove when we have dqlite watchers on k8s
+		w.logger.Warningf("error subscribing to namespace %q: %v", w.namespace, err)
+		subscription = noopSubscription{}
```

Comment:
This in theory should be fixed in main, but we'll have to see. This should be removed as a priority. I'll have to look into why this is happening if it's not already fixed.